app/controllers/map_controller.rb

```diff
@@ -16,7 +16,10 @@ def index
   end
 
   def show
-    @node = DrupalNode.find_map_by_slug(params[:name]+'/'+params[:date])
+    @node = DrupalNode.find params[:name]
```

Comment:
Gosh, I'm not sure what to do, then. But in the research notes, you were
able to not include the `/username/` in the slug, though it was in the URL.
Is nothing similar possible here, because the date comes *after* the slug?

On Sun, Jul 24, 2016 at 1:05 AM, Ananya Maiti <notifications@github.com>
wrote:

> In app/controllers/map_controller.rb
> <https://github.com/publiclab/plots2/pull/600#discussion_r71981928>:
>
> > @@ -16,7 +16,10 @@ def index
> >    end
> >
> >    def show
> > -    @node = DrupalNode.find_map_by_slug(params[:name]+'/'+params[:date])
> > +    @node = DrupalNode.find params[:name]
>
> @jywarren <https://github.com/jywarren> Actually friendly_id
> parameterizes the whole string. Slashes don't get well with parameterize.
> They get converted to dashes. Do you have any other format in mind? Though
> you can always use the default behavior friendly_id uses that appends a
> unique index at the en of slugs with similar title.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/publiclab/plots2/pull/600/files/c968e35c5fa7b3232e5e5b2ca0bc9afa658586cd#r71981928>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABfJ1TA4viSJGypXFtMIis0BCIsvZr5ks5qYvJ_gaJpZM4JDL2B>
> .
>